src/main/java/io/lettuce/core/LcsArgs.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2011-Present, Redis Ltd. and Contributors
+ * All rights reserved.
+ *
+ * Licensed under the MIT License.
+ *
+ * This file contains contributions from third-party contributors
+ * licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.lettuce.core;
+
+import io.lettuce.core.internal.LettuceAssert;
+import io.lettuce.core.protocol.CommandArgs;
+
+/**
+ * Argument list builder for the Redis <a href="https://redis.io/commands/lcs">LCS</a> command. Static import the methods from
+ * {@link LcsArgs.Builder} and call the methods: {@code keys(…)} .
+ * <p>
+ * {@link LcsArgs} is a mutable object and instances should be used only once to avoid shared mutable state.
+ *
+ * @author Dltmd202
+ * @since 7.0
```

Comment:
Let's put 6.6 here, It tihnk we can publish this in the next minor release

Reply:
Got it. I'll change all the @since tags in the code I uploaded to 6.6!